File: backend/src/main/java/com/qme/qmeapp/queue/dto/JoinQueueResponse.java
package com.qme.qmeapp.queue.dto;

import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class JoinQueueResponse {
    private UUID entryId;
    private Integer queueNumber;
    private Integer positionInLine;
    private String status;
    private Integer estimatedWaitTimeMin;
    private Integer estimatedWaitTimeMax;
}

File: mobile/app/src/main/java/com/qme/mobile/data/model/response/JoinQueueResponse.kt
package com.qme.mobile.data.model.response

data class JoinQueueResponse(
    val entryId: String,
    val queueNumber: Int,
    val positionInLine: Int,
    val status: String,
    val estimatedWaitTimeMin: Int?,
    val estimatedWaitTimeMax: Int?
)
